common/views/components/Caption/Caption.tsx

```diff
@@ -26,20 +26,23 @@ type Props = {
   caption: prismicT.RichTextField;
   preCaptionNode?: ReactNode;
   width?: number;
+  isFigcaption?: boolean;
 };
 
 const Caption: FunctionComponent<Props> = ({
   caption,
   preCaptionNode,
   width,
+  isFigcaption = true,
 }: Props) => {
   return (
     <Space
       v={{
         size: 'm',
         properties: ['margin-top'],
       }}
-      as="figcaption"
+      as={isFigcaption ? 'figcaption' : undefined}
+      aria-hidden={isFigcaption ? undefined : 'true'}
```

Comment:
Guess in this case it could be 
`...(isFigcaption && { aria-hidden: true, as: 'figcaption' })`

Reply:
I'm not sure what I'm doing wrong, but I couldn't make this syntax work. Going to merge as is and make a note to pick your brain next week.

Reply:
I might've written it wrong, did it from memory 😅  but yeah let's :) !

Reply:
I think I probably need wrapping braces and quotes around 'aria-hidden' maybe?

`{...(isFigcaption && { 'aria-hidden': true, as: 'figcaption' })}`

Reply:
Yeah makes sense, right? Found an example here :) https://github.com/wellcomecollection/wellcomecollection.org/blob/main/common/views/components/Header/Header.tsx#L358

Reply:
Actually this might be a better example https://github.com/wellcomecollection/wellcomecollection.org/blob/main/common/views/components/Pagination/Pagination.tsx#L90 (no need to return an empty object, unless it's a required field)

Reply:
yeah absolutely, sorry that's on me, I should've had a disclaimer than I meant "something like this" or just do it properly 😄  but yeah, they're object keys so i it would need to be a string 👍